actor/builtin/miner/miner.go

```diff
@@ -449,6 +458,29 @@ func (ma *Actor) GetSectorCommitments(ctx exec.VMContext) (map[string]types.Comm
 	return a, 0, nil
 }
 
+// GetSectorSize returns the size of the sectors committed to the network by
+// this miner.
+func (ma *Actor) GetSectorSize(ctx exec.VMContext) (*types.BytesAmount, uint8, error) {
```

Comment:
BLOCKING: This method needs tests.

Reply:
Done. See earlier gripe about `MinerGetSectorSize` test.